src/web/tools/GestureHandlerOrchestrator.ts

```diff
@@ -89,8 +95,13 @@ export default class GestureHandlerOrchestrator {
   public onHandlerStateChange(
     handler: GestureHandler,
     newState: State,
-    oldState: State
+    oldState: State,
+    forceStateChange?: boolean
```

Comment:
```suggestion
    sendIfDisabled?: boolean
```
I'd name it to be more descriptive, as it's used only for sending events to disabled handlers. (I'm not really convinced of the one I've proposed in case you have better ideas)